src/cr/cube/stripe/cubemeasure.py

```diff
@@ -92,6 +124,15 @@ def means(self):
         return self._means
 
 
+class _CatCubeOverlaps(_BaseCubeOverlaps):
+    """Overlaps cube-measure for a non-MR stripe."""
```

Comment:
Out of curiosity: do we have overlaps for non MR stripe? Considering that this is a 1D case (because of the name Stripe :D) why we need a `CatCubeOverlap` case?

Reply:
don't need it, removed it...